python/mxnet/ndarray/numpy/_op.py

```diff
@@ -1734,13 +1734,13 @@ def histogram(a, bins=10, range=None, normed=None, weights=None, density=None):
     if isinstance(bins, numeric_types):
         if range is None:
             raise NotImplementedError("automatic range is not supported yet...")
-        return _npi.histogram(a, bin_cnt=bins, range=range)
+        return _api_internal.histogram(a, None, bins, range)
```

Comment:
`tuple(_api_internal.histogram(a, None, bins, range))` may be better. Otherwise the `ADT` is directly returned to users.

Reply:
Done